src/components/performance/late-icon.cjsx

```diff
@@ -0,0 +1,22 @@
+React    = require 'react'
+BS       = require 'react-bootstrap'
+S = require '../../helpers/string'
+moment = require 'moment'
+
+module.exports = React.createClass
+  displayName: 'LateIcon'
+  propTypes:
+    task: React.PropTypes.shape(
+      due_at:          React.PropTypes.string
+      last_worked_at:  React.PropTypes.string
+      type:            React.PropTypes.string
+    ).isRequired
+
+  render: ->
+    due = moment(@props.task.due_at)
+    msg = S.capitalize(@props.task.type) + ' was worked ' + due.from(@props.task.last_worked_at, true) + ' late'
+
```

Comment:
Hm.  How about an exported method(s) in a `helpers/task` file?  That way both could share it?
It would take an object with `type`, `last_worked_at` and `due_at` fields and return the same at the existing `getStepLateness` ?